sdk/python/feast/repo_config.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import NamedTuple, Optional
+
+import yaml
+from bindr import bind
+
+
+class LocalOnlineStoreConfig(NamedTuple):
+    path: str
+
+
+class OnlineStoreConfig(NamedTuple):
+    local: Optional[LocalOnlineStoreConfig] = None
+
+
+class RepoConfig(NamedTuple):
```

Comment:
What if we just called this `Config`?

Reply:
I think we already have a Config or two so I wanted to be a bit more specific here

Reply:
We should reconcile https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_store_config.py and this since I think they're the same thing